pom.xml

```diff
@@ -342,6 +355,12 @@
 			<artifactId>webauthn-server-core</artifactId>
 			<version>${yubico.version}</version>
 		</dependency>
+		<dependency>
```

Comment:
do you really need this additional dependency?

Reply:
No, the dependency is not required and was a leftover from previous version of the code. To be removed in next commit.